src/main/java/com/minecolonies/coremod/entity/ai/mobs/EntityAIWalkToRandomHuts.java

```diff
@@ -0,0 +1,162 @@
+package com.minecolonies.coremod.entity.ai.mobs;
+
+import com.minecolonies.api.util.Log;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.buildings.AbstractBuilding;
+import com.minecolonies.coremod.entity.pathfinding.GeneralEntityWalkToProxy;
+import com.minecolonies.coremod.entity.pathfinding.PathNavigate;
+import net.minecraft.entity.EntityCreature;
+import net.minecraft.entity.EntityLiving;
+import net.minecraft.entity.ai.EntityAIBase;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.lang.reflect.Field;
+import java.util.Collection;
+import java.util.Random;
+
+/**
+ * Created by Asher on 7/6/17.
+ */
+public class EntityAIWalkToRandomHuts extends EntityAIBase
+{
+
+    protected final EntityCreature entity;
+    private         BlockPos       targetBlock;
+    protected final World          world;
+    protected final double         speed;
+    protected final Colony         colony;
+
+    /**
+     * Walk to proxy.
+     */
+    private GeneralEntityWalkToProxy proxy;
+
+    /**
+     * The navigator for this entity.
+     */
+    final private PathNavigate newNavigator;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)